doc/reference/power_management/index.rst

```diff
@@ -447,9 +486,19 @@ Suspend Device asynchronously API
 
    int pm_device_put(const struct device *dev);
 
-Marks the device as being released. This API asynchronously put
-the device to suspend state if not already in suspend state.
-The API returns 0 on success.
+Releases a device. This API asynchronously puts the device to suspend
+state if not already in suspend state if the usage count of this device
+reaches 0.
+
+Applications can monitor this operation waiting on the device
+condition variable :code:`dev->pm->condvar`. e.g:
+
+.. code-block:: c
+
```

Comment:
Applications should not be dealing with condition variables at all, this is all done on the device driver/device pm level.

Reply:
this was not addressed.

Reply:
It was ... I replaced `applications` with`device drivers`.

```
Device drivers can monitor this operation waiting on the device
condition variable :code:`dev->pm->condvar`. e.g:
```